lib/article-fullpage/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component, PropTypes } from 'react';
+
+require('./style.css');
+
+export default class ArticleFullPage extends Component {
+  render () {
+    const { articleContent } = this.props;
+    return (
+        <div className='articleFullPageContainer'>
```

Comment:
just watch out for the indentation here - think there is an extra space for all the elements inside the return